docs/diagrams/sort/style.puml

```diff
@@ -0,0 +1,79 @@
+/'
```

Comment:
i think it is better to just use the ` styles.puml` in docs/diagrams. no point replicating it in the subfolder

Reply:
Thanks